### ISSUE TITLE
[DOC] Type of "metrics" of deep learning classifiers using `keras` could be improved

### ISSUE THREAD

**Issue by dschrempf** [aeon-toolkit/aeon#2435]
### Describe the issue linked to the documentation

For example, the `metrics` argument of the `LITETimeCLassifier` (https://www.aeon-toolkit.org/en/stable/api_reference/auto_generated/aeon.classification.deep_learning.LITETimeClassifier.html#aeon.classification.deep_learning.LITETimeClassifier) states:

> metricsstr or list[str], default=”accuracy”
> The evaluation metrics to use during training. If a single string metric is provided, it will be used as the only metric. If a list of metrics are provided, all will be used for evaluation.

However, one can (and has to, if using more complicated metrics) directly use the Keras classes. For example,
```python
        metrics=[
            keras.metrics.AUC(),
            keras.metrics.FBetaScore(average="macro", beta=4.0),
        ],
```

### Suggest a potential alternative/fix

Keras uses the following wording:
> List of metrics to be evaluated by the model during training and testing. Each of this can be a string (name of a built-in function), function or a [keras.metrics.Metric](https://keras.io/api/metrics/base_metric#metric-class) instance.

**Comment by TonyBagnall** [comment]
thanks for that, would welcome a PR
